qcodes_contrib_drivers/drivers/Bilt/ITest.py

```diff
@@ -0,0 +1,467 @@
+        """
+        Set output voltage range
+        
+        Args:
+            chan : The 1-indexed channel number
+            volt : Voltage range (1.2 or 12)
+        """
+        chan_id = self.chan_to_id(chan)
+        self.write(chan_id + 'VOLT:RANGE ' + str(volt))
+
+
+    def _get_chan_range(self, chan:int) -> str:
+        """
+        Get output voltage range
+        
+        Args:
+            chan: The 1-indexed channel number
+            
+        Returns:
+            volt: Output voltage range
+        """
+        chan_id = self.chan_to_id(chan)
+        
+        return self.ask(chan_id + 'VOLT:RANGE?')[:-2]
+
+
+    def _set_chan_pos_sat(self, chan:int,
+                                pos_sat: Union[int, str]) -> None:
+        chan_id = self.chan_to_id(chan)
+        if isinstance(pos_sat,(int,float)):
+            self.write(chan_id + 'VOLT:SAT:POS {:.8f}'.format(pos_sat))
+        elif isinstance(pos_sat,str):
+            self.write(chan_id + 'VOLT:SAT:POS MAX')
+
+
+    def _set_chan_neg_sat(self, chan:int,
+                                neg_sat: Union[int, str]) -> None:
+        chan_id = self.chan_to_id(chan)
+        if isinstance(neg_sat,(int,float)):
+            self.write(chan_id + 'VOLT:SAT:NEG {:.8f}'.format(neg_sat))
+        elif isinstance(neg_sat,str):
+            self.write(chan_id + 'VOLT:SAT:NEG MIN')
+
+
+    def _get_chan_pos_sat(self, chan:int) -> str:
+        chan_id = self.chan_to_id(chan)
+        return self.ask(chan_id + 'VOLT:SAT:POS ?')
+
+
+    def _get_chan_neg_sat(self, chan:int) -> str:
+        chan_id = self.chan_to_id(chan)
+        return self.ask(chan_id + 'VOLT:SAT:NEG ?')
+
+
+    def _get_chan_state(self, chan:int) -> bool:
+        """
+        Get channel power state
+        
+        Args:
+            chan: The 1-indexed channel number
+            
+        Returns:
+            state: Power state 
+        """
+        chan_id = self.chan_to_id(chan)
+        state = self.ask(chan_id + 'OUTP ?')
+        
+        if state == '1':
+            return True
+        elif state == '0':
+            return False
+
+
+    def _set_chan_state(self, chan:int,
+                              state: bool) -> None:
+        """
+        Set channel power state
+        
+        Args:
+            chan: The 1-indexed channel number
+            state: power state
+        """
+        chan_id = self.chan_to_id(chan)
+        if state:
+            state_str = 'on'
+        else:
+            state_str = 'off'
+        self.write(chan_id + 'OUTP ' + state_str)
+
+
+    def _set_chan_name(self, chan:int,
+                             name: str) -> None:
+        """
+        Set the name of the channel
+
+        Args:
+            chan: Channel to be named
+            name: Name of the channel
+        """
+        chan_id = self.chan_to_id(chan)
+        self.write(chan_id + 'chan:name "{}"'.format(name))
+
+
+    def chan_to_ic(self, chan:int) -> Tuple[int]:
```

Comment:
What are the types of 'i' and 'c'?  I think 'i' is a float and 'c' is an int. If yes, then the return type should be Tuple[float, int]

Reply:
Corrected.